server/sequencer_manager.go

```diff
@@ -37,6 +39,7 @@ func NewSequencerManager(registry extension.Registry, gw time.Duration) *Sequenc
 	return &SequencerManager{
 		guardWindow: gw,
 		registry:    registry,
+		signers:     make(map[int64]*crypto.Signer),
```

Comment:
Should we protect this with a mutex, or are Sequencers not meant to be thread safe? If the latter, please document.

I believe this is tied to the Travis failure as well.